security-analysis/src/test/java/eu/itesla_project/security/json/JsonConverterTest.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.security.json;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import com.fasterxml.jackson.core.JsonGenerationException;
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.core.util.DefaultPrettyPrinter;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+
+import eu.itesla_project.contingency.BranchContingency;
+import eu.itesla_project.contingency.Contingency;
+import eu.itesla_project.contingency.ContingencyElement;
+import eu.itesla_project.contingency.ContingencyImpl;
+import eu.itesla_project.contingency.GeneratorContingency;
+import eu.itesla_project.iidm.network.Country;
+import eu.itesla_project.iidm.network.Generator;
+import eu.itesla_project.iidm.network.Line;
+import eu.itesla_project.iidm.network.Network;
+import eu.itesla_project.iidm.network.test.EurostagTutorialExample1Factory;
+import eu.itesla_project.security.LimitViolation;
+import eu.itesla_project.security.LimitViolationType;
+import eu.itesla_project.security.PostContingencyResult;
+import eu.itesla_project.security.PreContingencyResult;
+import eu.itesla_project.security.SecurityAnalysisResult;
+
+/**
+ *
+ * @author Massimo Ferraro <massimo.ferraro@techrain.it>
+ */
+public class JsonConverterTest {
+    
+    private Network network;
+    private SecurityAnalysisResult result;
+    private String jsonResult;
+    private String ppJsonResult;
+    
+    @Before
+    public void setUp() throws JsonParseException, JsonMappingException, IOException {
+        network = EurostagTutorialExample1Factory.create();
+        Line line1 = network.getLine("NHV1_NHV2_1");
+        Line line2 = network.getLine("NHV1_NHV2_2");
+        Generator generator = network.getGenerator("GEN");
```

Comment:
There is no need to create a full network to create a SecurityAnalysisResult once LimitViolation.subject replacement is done.